docs/tutorials/java/ssd_inference.md

```diff
@@ -0,0 +1,182 @@
+# Multi Object Detection using pre-trained SSD Model via Java Inference APIs
+
+This tutorial shows how to use MXNet Java Inference APIs to run inference on a pre-trained Single Shot Detection (SSD) Model.
+
+The SSD model is trained on the Pascal VOC 2012 dataset. The network is a SSD model built on Resnet50 as base network to extract image features. The model is trained to detect the following entities (classes): ['aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus', 'car', 'cat', 'chair', 'cow', 'diningtable', 'dog', 'horse', 'motorbike', 'person', 'pottedplant', 'sheep', 'sofa', 'train', 'tvmonitor']. For more details about the model, you can refer to the [MXNet SSD example](https://github.com/apache/incubator-mxnet/tree/master/example/ssd).
+
+## Pre-Requisites
+
+To complete this tutorial, we need : 
+* [MXNet Java Setup on IntelliJ IDEA](/java/mxnet_java_on_intellij.html) 
```

Comment:
Can't you do this tutorial using another IDE?

Reply:
I have provided the users a way to run it using command line as well at the bottom of this tutorial. 
Given that the setup instructions for Scala are also based on IntelliJ, I wanted to follow the same pattern for consistency. 
There's nothing stopping a user from using any other compatible IDE other than IntelliJ.

Reply:
I suggest to mark it optional.

Reply:
Done.